### PR TITLE
Updated comments in docker-compose.yml

### PR DESCRIPTION
**Pull request by pierrealixt**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -18,7 +17,7 @@ services:
       - POSTGRES_USER=docker
       - POSTGRES_PASS=docker
       - ALLOW_IP_RANGE=0.0.0.0/0
-      # Add extensions you need to be enabled by default in the DB. Default are the three specified below
+      # Add extensions you need to be enabled by default in the DB. Default are the six specified below
```
Comment (NyakudyaA) [inline docker-compose.yml]: @pierrealixt We should remove the last three `pointcloud,pointcloud_postgis` and mention that users can also install these three on startup

Comment (pierrealixt) [reply docker-compose.yml]: in the README or only as a simple comment in the docker-compose file?

Comment (NyakudyaA) [reply docker-compose.yml]: In this README. Just add a comment on this section https://github.com/kartoza/docker-postgis#build-with-custom-extensions. 

Something like this
The image is currently built to accommodate the following extensions `pointcloud,pointcloud_postgis` which you can use by with  the environment variable
POSTGRES_MULTIPLE_EXTENSIONS

**Comment by NyakudyaA** [comment]
Otherwise LGTM @pierrealixt

**Pull request merged**